docs/reference/traced_probes.md

```diff
@@ -0,0 +1,292 @@
+# TRACED_PROBES(8)
+
+## NAME
+
+traced_probes - System & OS Probes
+
+## DESCRIPTION
+
+`traced_probes` is a specialized daemon that acts as a privileged
+[Producer](/docs/concepts/service-model.md#producer) in the Perfetto
+architecture. While any application can act as a producer to contribute its own
+trace data, `traced_probes` is specifically responsible for collecting
+system-level and kernel-level data that typically requires elevated privileges.
+
+## Relationship with `traced`
+
+`traced_probes` is a client of the [`traced`](/docs/reference/traced.md)
+service. It connects to `traced`'s producer socket and registers a set of data
+sources. `traced` then sends requests to `traced_probes` to start or stop these
+data sources as part of a tracing session.
+
+This separation of concerns is a key part of Perfetto's design. `traced` is the
+central manager, while `traced_probes` is a specialized data provider. This
+decoupled architecture allows for multiple, independent producers and consumers
+to interact with the tracing system simultaneously without interfering with each
+other.
+
+![traced_probes and traced](/docs/images/platform-tracing.png)
+
+## Data Sources
```

Comment:
I would strongly suggest we separate out the android data source from the Linux ones to make it very clear.

Reply:
Also we should really give a snippet of the config as an example so people don't have to go searching for it.